app_dart/lib/src/request_handlers/check_for_waiting_pull_requests_queries.dart

```diff
@@ -8,7 +8,7 @@ query LabeledPullRequcodeestsWithReviews($sOwner: String!, $sName: String!, $sLa
     labels(first: 1, query: $sLabelName) {
       nodes {
         id
-        pullRequests(first: 100, states: OPEN) {
+        pullRequests(first: 100, states: OPEN, orderBy: {direction: ASC, field: CREATED_AT}) {
```

Comment:
Not sure how to test this other than asserting the query contains it, which seems excessive.